app/views/passwordless/sessions/create.html.erb

```diff
@@ -0,0 +1,9 @@
+<% content_for :class, "login-panel--centred"%>
+
+<%= image_tag "green-tick.svg", alt: "success" %>
+
+<h2>Check your email</h2>
+
+<p>If <strong>firstname.lastname@gmail.com</strong> exists in our records, you'll get an email with a special link.</p>
```

Comment:
Did you mean to use the email address the user entered here, rather than a hardcoded gmail address? Also, while we're at it, we may want to say, "If you don't get an email, speak to _xyz_" or that "you may not have an account setup by an administrator yet."

Reply:
i did some more looking and figured out how to fish out the email and show it on this screen